src/styles/document-export-page.css

```diff
@@ -167,6 +167,14 @@
 .joubel-create-document .header-buttons button {
   display: flex;
   align-items: center;
+  background-color: var(--h5p-theme-main-cta-base);
+  color: var(--h5p-theme-contrast-cta);
+}
+
+.joubel-create-document .header-buttons button:hover {
+  background-color: var(--h5p-theme-main-cta-light);
+  border-color: var(--h5p-theme-main-cta-light);
+  color: var(--h5p-theme-contrast-cta);
```

Comment:
this should be added up in line 143~ 

<img width="704" height="132" alt="Image" src="https://github.com/user-attachments/assets/e2c6bdf4-f537-43ef-8fa4-0226669d62cb" />

Reply:
https://github.com/h5p/h5p-document-export-page/pull/68 to resolve, cannot push to H5P Group repositories anymore.